docs/src/modules/java/pages/value-entity.adoc

```diff
@@ -107,29 +107,3 @@ As our example relies on the code generation, the registration gets automaticall
 ----
 include::java:example$valueentity-counter/src/main/java/com/example/Main.java[]
 ----
-
-== Testing the Entity
```

Comment:
I needed to remove this part because the codegen is temporarily not generating the unit test. 

This section will need to be re-written anyway because it should cover the upcoming testkit.